crates/accelerate/src/sabre_layout.rs

```diff
@@ -39,7 +39,7 @@ pub fn sabre_layout_and_routing(
     num_swap_trials: usize,
     num_layout_trials: usize,
     seed: Option<u64>,
-) -> ([NLayout; 2], (SwapMap, PyObject, NodeBlockResults)) {
+) -> (NLayout, Vec<usize>, (SwapMap, PyObject, NodeBlockResults)) {
```

Comment:
Yeah probably, I was just being lazy because it's a relatively small vector and I intended to completely revisit this after this PR.  tbh, I probably should have left this as `NLayout` and done the conversion to a permutation in Python space to keep the diff down, but I think I got all confused dealing with the disjoint components, and thought I needed to do it in Rust.  Oh well.

Fixed the type in 66dbdea8a.